projects/demo/src/app/demo/pages/table-page/table-sortable-header-page/table-sortable-header-page.component.html

```diff
@@ -33,6 +35,25 @@
                 </pa-table-lead-cell-multi-line>
             </pa-table-row>
         </pa-table>
+
+        <p>Header staring with a checkbox column</p>
```

Comment:
typo => "starting" not "staring" (I assume)

Reply:
good catch